tests/runtests/aux/lib.sh

```diff
@@ -121,6 +121,17 @@ function generate_python3_exception() {
     su -c will_python3_raise $1
 }
 
+function generate_crash_unpack() {
+    rlLog "Generate unpackaged crash"
```

Comment:
Maybe, change directory to a safe directory like /tmp.

Reply:
yes, you're right, good point, thank you